modules/src/ics04_channel/msgs.rs

```diff
@@ -39,10 +39,47 @@ pub enum ChannelMsg {
     ChannelCloseConfirm(MsgChannelCloseConfirm),
 }
 
+/*
+    `PacketMsg` represents packet messages sent between an ALPHA chain
```

Comment:
> I'm not sure the right place to document these are in the modules (as we're doing here).

Yeah, I think we can put all explanation in one place, or cross reference to other places where it is used. The documentation should also target cargo doc so that readers can read from the generated Rustdoc.

> The exact details of how packets flow depend quite a bit on the relayer's internal bookkeeping: a RecvPacket could equally be flowing from what we call a source to a destination, or the other way around.

Here I try to persist the chain roles across module boundaries. So for example if a `RelayPath` object treats its source chain as Alpha and its destination chain as Beta, then my understanding is that the same `RelayPath` should not suddenly treat its source chain as Beta and its destination chain as Alpha. If my assumption is wrong, then it is even more important to determine where the roles are flipped.